locales/en/events.json

```diff
@@ -0,0 +1,412 @@
+    "description": "{{supplier_personnel_numbers, oxfordJoin}} reported an assault event at {{reported_at, formatTime}} at {{location.title}}<br>$t(events::select_fault_classification)<br>$t(events::select_incident_vehicle)"
+  },
+  "PersonMoveDeathInCustody": {
+    "heading": "Death in custody",
+    "description": "{{supplier_personnel_numbers, oxfordJoin}} reported a death in custody at {{reported_at, formatTime}} at {{location.title}}<br>$t(events::select_fault_classification)<br>$t(events::select_incident_vehicle)"
+  },
+  "PersonMoveBookedIntoReceivingEstablishment": {
+    "heading": "{{person.fullname}} booked into {{location.title}}",
+    "description": ""
+  },
+  "PersonMoveMajorIncidentOther": {
+    "heading": "Major incident occurred",
+    "description": "$t(events::incident_level_details, {\"incident_level\": \"Major\"})"
+  },
+  "PersonMoveMinorIncidentOther": {
+    "heading": "Minor incident occurred",
+    "description": "$t(events::incident_level_details, {\"incident_level\": \"Minor\"})"
+  },
+  "PersonMovePersonEscapedKpi": {
+    "heading": "KPI Escape occured",
+    "description": "$t(events::incident_escaped)"
+  },
+  "PersonMovePersonEscaped": {
+    "heading": "Non KPI Escape occured",
+    "description": "$t(events::incident_escaped)"
+  },
+  "PersonMoveReleasedError": {
+    "heading": "Released in error",
+    "description": "{{person.fullname}} was released in error from {{location.title}} $t(events::incident_reported_classified)"
+  },
+  "PersonMoveRoadTrafficAccident": {
+    "heading": "Road traffic accident",
+    "description": "Vehicle {{vehicle_reg}} transporting {{person.fullname}} met with road traffic accident $t(events::incident_reported_classified)"
+  },
+  "PersonMoveSeriousInjury": {
+    "heading": "Serious injury sustained",
+    "description": "{{person.fullname}} sustained serious injury $t(events::select_incident_location_preposition) $t(events::select_incident_location_vehicle) $t(events::incident_reported_classified)"
+  },
+  "PersonMoveUsedForce": {
+    "heading": "Supplier used force",
+    "description": "{{move.supplier.name}} staff {{supplier_personnel_numbers}} used force on {{person.fullname}} <br>The incident was reported at {{reported_at, formatTime}} <br>$t(events::select_fault_classification)"
+  },
+  "PersonMoveVehicleBrokeDown": {
+    "heading": "Supplier vehicle breakdown",
+    "description": "Vehicle {{vehicle_reg}} transporting {{person.fullname}} broke down $t(events::incident_reported_classified)"
+  },
+  "PersonMoveVehicleSystemsFailed": {
+    "heading": "Supplier vehicle systems failed",
+    "description": "Vehicle {{vehicle_reg}} transporting {{person.fullname}} has had a system failure $t(events::incident_reported_classified)"
+  },
+  "StatusMoveBooked": {
+    "statusChange": true,
+    "heading": "Booked",
+    "description": "Move has been booked with {{move.supplier.name}}"
+  },
+  "StatusMoveCancelled": {
+    "statusChange": true,
```

Comment:
I think this is another thing that stands out that suggests we're using translations for more than they should be. 

I understand you are under time pressures to deliver this work but it feels like a cleaner approach would be to define the event "types" somewhere else (maybe in the presenter itself) that are classed as "status changes".

Reply:
The more I do think about these "status" events, I think it would make more sense to use the translations we've already defined within [`statuses`](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/locales/en/statuses.json). That also ensures we keep consistent language.

I'll post a suggestion in the presenter.

Reply:
On second thoughts, this might be a larger change. 

Does the API by any chance show the previous/current status as a separate value when these event types are returned?

Could we not just look at the "type" for something like a status change that returns the current value? That way the presenter could make sure of the existing keys.

Failing that, could we re-use the values here so that we are consistent across the system:

```json
  "StatusMoveCompleted": {
    "statusChange": true,
    "heading": "$(statuses::completed)",
    "description": "Move has been completed by {{move.supplier.name}}"
  },
```

Reply:
Parsing all that and taking away the point that we should be doing

`"heading": "$t(statuses::completed)"`

I agree in principle. Just getting it checked off

Reply:
👍   Event statuses now aliased to the corresponding status strings